index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en-gb">
+<head>
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link href="normalize.css" type="text/css" rel="stylesheet" />
+    <!-- Add character set when decided -->
```

Comment:
placeholder not required, charset already declared and font will be added when needed